metabulo/analyses.py

```diff
@@ -0,0 +1,48 @@
+from itertools import combinations
+from typing import Any, Dict, List, Optional, Union
+
+import pandas as pd
+from scipy.stats import wilcoxon
+
+from .opencpu import opencpu_request
+
+
+def wilcoxon_test(measurements: pd.DataFrame, zero_method: Optional[str] = None,
+                  alternative: Optional[str] = None) -> Dict[str, Union[List[str], List[Dict]]]:
+
+    if zero_method is None:
+        zero_method = 'wilcox'
+    if alternative is None:
+        alternative = 'two-sided'
```

Comment:
cause I wanted to be None a valid parameter which simplifies when calling the method with the alternative parameter but with None since no value is given, e.g. from the web request